src/batched/dense/KokkosBatched_Gemm_Decl.hpp

```diff
@@ -358,13 +452,63 @@ int BatchedGemm(const BatchedGemmHandleType *handle, const ScalarType alpha,
                                         Algo::Gemm::Blocked>::type>::type>::
           type;
 
-      // TODO: Add additional heuristic conditionals that look at M,N to
-      // determine
-      //       whether to invoke BatchedSerialGemm, BatchedDblBuf, etc.
-      ret = BatchedSerialGemm<ArgTransA, ArgTransB, bsgModeType, ArgBatchSzDim,
-                              bsgResultsPerThread, ScalarType, AViewType,
-                              BViewType, CViewType>(alpha, A, B, beta, C)
+      // if (on_gpu && c_m >= 20 &&
```

Comment:
Does this need to be kept?

Reply:
No, it is just a placeholder for invoking another algo that moves entire tiles into shared memory before GEMM.